IFComp/root/src/about/file_formats.tt

```diff
@@ -61,9 +61,15 @@
 
 <h3 id="adrift">ADRIFT games</h3>
 
-<p><strong>File extension:</strong> <code>.blorb</code></p>
+<p><strong>File extensions:</strong> <code>.taf</code>, <code>.blorb</code></p>
 
-<p>Games made with <a href="http://www.adrift.co">ADRIFT</a> also have a "blorb" file extension, just like <a href="#inform">games made with Inform</a>.</p>
+<p><a href="http://www.adrift.co">ADRIFT</a> is another system for creating parser-based IF. Like TADS and Inform, playing its games requires the use of <a href="http://www.adrift.co/files/ADRIFT5r.zip">a separate interpreter ("runner") program</a>, available for Windows.</p>
+
+<p>(Linux/Mac users: An alternate Linux/Mono interpreter is also available on the <a href="http://www.adrift.co/download">ADRIFT downloads page</a>, but current reports as of 2018 indicate that it doesn't work.)</p>
+
+<p>You can also play ADRIFT games on Android using the <a href="https://play.google.com/store/apps/details?id=com.luxlunae.fabularium">Fabularium</a> interpreter.</p>
+
+<p>Note that games made with ADRIFT have a "blorb" file extension, just like <a href="#inform">games made with Inform</a>, but they don't work with Lectrote, Gargoyle, or other popular IF interpreters.</p>
```

Comment:
Edit this a bit given the existence of ".taf" files, please. _games with ADRIFT can have a "blorb" file extension_, e.g.

Reply:
done